site/_includes/explanations/partitioned-topic-admin.md

```diff
@@ -1,4 +1,4 @@
-You can use Pulsar's [admin API](../../admin/AdminInterface) to create and manage partitioned topics.
+You can use Pulsar's [admin API](../../admin-api/overview) to create and manage partitioned topics.
```

Comment:
Oh, I think I misunderstood your comment then. 
Are you saying to add the link to `/docs/latest/advanced/PartitionedTopics` ? Can we do it later though? Let's just fix the 404 errors for now.